.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,30 @@
+name: Feature Request
+description: Suggest a feature or idea to Rancher Desktop
+title: "[Feature Request]- "
+labels: "enhancement :bulb:"
```

Comment:
Can the label just be `enhancement` so that it matches the existing label we are already using.

Reply:
Sure thing.
Removed the emoji from the label.
When the user file an issue or enhancement, it should auto include the label on the body.

_labels (Array or String): This issue will automatically receive these labels upon creation. Can be array of labels or comma-delimited string, e.g. "bug,needs-triage"_

Thanks for review